docs/source/troubleshooting/index.rst

```diff
@@ -101,16 +101,71 @@ If this happens, then you should follow these remediation steps:
    This process should only be attempted if you are sure that the process has died, being aware that it may be running on another machine.
    Releasing the lock of a running process would invalidate the consistency guarantees of any backups that are started while that process is still running!
 
+Clear with cURL
+---------------
+
 1. Find the currently-held lock, by examining the logs. Attempting to acquire a lock will cause the currently held lock to be logged:
 
 .. code-block:: bash
 
   INFO  [2017-02-01 16:40:34,333] com.palantir.atlasdb.persistentlock.CheckAndSetExceptionMapper: Request failed.
-    Stored LockEntry: LockEntry{rowName=BackupLock, lockId=1e3a8db1-a1a6-4aae-96bd-e3107b709c5e, reason=manual-backup}
-
+    Stored persistent lock: LockEntry{lockName=BackupLock, instanceId=427eb02a-f017-40cd-8d08-0a163315029a, reason=manual-backup}
 
 2. Curl the ``release`` endpoint. Note that the required formatting is slightly different (keys and values must be surrounded with ``"``).
 
 .. code-block:: bash
 
-   $ curl -X POST --header 'content-type: application/json' '<product-base-url>/persistent-lock/release' -d '{"rowName":"BackupLock","lockId":"1e3a8db1-a1a6-4aae-96bd-e3107b709c5e","reason":"manual-backup"}'
+   $ curl -X POST --header 'content-type: application/json' '<product-base-url>/persistent-lock/release' -d '"427eb02a-f017-40cd-8d08-0a163315029a"'
+
+Clear with CQL (for Cassandra)
```

Comment:
I think we should remove the (for Cassandra) part because it might actually add confusion. In the "Clearing the schema mutation lock" section above, we just have "Clear with CQL", and now as a reader it's unclear if this CQL is different from the above CQL. I think the "tip" section you have clarifies that this is Cassandra specific.